postprocessing_variant_calls/maf/tag/tag_process.py

```diff
@@ -268,5 +268,171 @@ def traceback(
     return 0
 
 
+# @app.command(
```

Comment:
reminder to cleanup comments and TODOS

Reply:
7ab98306a2b85867c65c4923919d82f784a9165f